### DISCUSSION TITLE
How to use "configuration-discovery.default.agentConfigurations" control global?

### DISCUSSION THREAD

**Discussion by peoplewhoworkhard** [apache/skywalking#10548] [Q&A] [answered]
I used nacos to dynamically configure the skywalking agent configuration. 

There is a configuration: configuration-discovery.default.agentConfigurations, 
Usage reference:
https://github.com/apache/skywalking-java/blob/20fb8c81b3da76ba6628d34c12d23d3d45c973ef/docs/en/setup/service-agent/java-agent/configuration-discovery.md
![图片](https://user-images.githubusercontent.com/44570840/225573206-c63127b5-950e-48bc-8d59-8803077b1e0b.png)

-----
I wonder how to control global configuration with this, thanks!

**Comment by pg-yang** [comment, marked as answer]
https://github.com/apache/skywalking/pull/10476 is processing
